<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PBMainWindowClass</class>
 <widget class="QWidget" name="PBMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Practice Bioinformatics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QListView" name="m_ProgramFileListLV">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_LoadProgramsBtn">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Load programs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="m_EditorTE">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="tabStopWidth">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="m_ConsoleTE">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="tabStopWidth">
          <number>40</number>
         </property>
         <property name="placeholderText">
          <string>&gt;&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_RunProgramBtn">
         <property name="text">
          <string>Run program</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/PBMainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_LoadProgramsBtn</sender>
   <signal>clicked()</signal>
   <receiver>PBMainWindowClass</receiver>
   <slot>SetFilesDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ProgramFileListLV</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>PBMainWindowClass</receiver>
   <slot>ReadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_RunProgramBtn</sender>
   <signal>clicked()</signal>
   <receiver>PBMainWindowClass</receiver>
   <slot>RunProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetFilesDirectory()</slot>
  <slot>ReadFile()</slot>
  <slot>RunProgram()</slot>
 </slots>
</ui>
